topic5/contracts/Service.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.4.23;
+
+contract Service {
+    
+    address public owner;
```

Comment:
Could reuse the already written Ownable contract

Reply:
Yup.